src/main/java/org/jboss/xavier/analytics/pojo/input/VMWorkloadInventoryModel.java

```diff
@@ -0,0 +1,179 @@
+package org.jboss.xavier.analytics.pojo.input;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+
+
+public class VMWorkloadInventoryModel
+{
+    //common/name
+    private String provider;
+    //common/ems_clusters/v_parent_datacenter
+    private String datacenter;
+    //common/ems_clusters/name
+    private String cluster;
+    //vms/name
+    private String vmName;
+    //sum of vms/hardware/disks/size_on_disk
+    private BigDecimal diskSpace;
+    //vms/ram_size_in_bytes
+    private Integer memory;
+    //vms/num_cpu
+    private Integer cpuCores;
+    //vms/operating_system/product_name
+    private String osProductName;
+    //hardware/guest_os_full_name
+    private String guestOSFullName;
+    //vms/has_rdm_disk
+    private boolean hasRdmDisk;
+    //count of nics object within the vms/hardware
+    private Integer nicsCount;
+    //hardware/disks/filename
+    private Collection<String> vmDiskFilenames;
+    private Collection<String> systemServicesNames;
+    private Collection<Map<String,String>> files;
```

Comment:
I feel like having just a Map (without the Collection) would be enough to accomplish what we decided during the review meeting.
Each file will be an entry with `key = file name` and `content = file content`.
In this way we should be able to collect all the files from the payload in a Map (without the "wrapping" Collection)

```suggestion
    private Map<String,String> files;
```